.env.e2e

```diff
@@ -0,0 +1,15 @@
+POINT_NODE_VERSION=dev
+BLOCKCHAIN_NETWORK_ID=256
+DEFAULT_BALANCE=0x64
+BLOCKCHAIN_HOST=blockchain_node
+BLOCKCHAIN_URL=blockchain_node:7545
+BLOCKCHAIN_PROVIDER_PORT=7545
+ACCOUNT_PK_AND_BALANCE="<YOUR_LOCAL_ACCOUNT_PK>,0x56bc75e2d63100000"
+DEV_ZAPP_HOST=<SET_THIS_TO_THE_ZAPP_HOST_NAME_YOU_ARE_DEVELOPING>
+DB_WEBSITE_OWNER=point_website_owner
+DB_WEBSITE_VISITOR=point_website_visitor
+DB_STORAGE_PROVIDER=point_storage_provider
+DB_TEST=point_test
+DB_USERNAME=pointuser
+DB_PASSWORD=pointpassword
+POINT_KEYSTORE="${HOME}/.point/keystore"
```

Comment:
Let's use `~` instead of `${HOME}` here (I'ver already changed it in other dotenv files). Otherwise there may be some problems when running on other OSs.